tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>The math is not rendering properly in the docs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I don't have a clue about what happened, maybe it's just a bug that will disappear by relaunching the building of the docs.</p>